docs/development/best-practices.md

```diff
@@ -3,6 +3,26 @@
 This document explains our best practices.
 Follow these best practices when you're working on our code.
 
+## Git branch names
+
+The Renovate maintainers checkout a lot of Git branches.
+When Git branches have names like `patch-1` it's hard to remember what each branch is about.
+You can help the maintainers by following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification when naming your Git branch.
```

Comment:
Do you think we can remove this, and get to the point?

Reply:
```suggestion
```

Yeah, let's remove the introduction and fast-forward to the instructions. I'm putting the link to the Conventional Commits spec to another sentence.